sympy/stats/tests/test_discrete_rv.py

```diff
@@ -1,15 +1,19 @@
-from sympy import S, Sum, I, lambdify, re, im, log, simplify, zeta, pi
-from sympy.abc import x
```

Comment:
why?

Reply:
`from sympy.abc import x` is slower than `x = Symbol('x')`